c/include/nvtx3/nvtx3.hpp

```diff
@@ -902,7 +915,7 @@ class domain {
    * "global" NVTX domain.
    *
    */
-  domain() noexcept {}
+  constexpr domain() noexcept {}
```

Comment:
This `constexpr` depends on C++11, and will also generate warnings on older versions of MSVC.

Reply:
Nevermind this point, I see we already use `constexpr` constructors throughout the file.